packages/graphile-build-pg/src/plugins/PgIntrospectionPlugin.js

```diff
@@ -199,6 +199,23 @@ export type PgEntity =
   | PgExtension
   | PgIndex;
 
+export type PgIntrospectionResultsByKind = {
+  __pgVersion: number,
+  attribute: PgAttribute[],
+  attributeByClassIdAndNum: { [string]: { [string]: PgAttribute } },
```

Comment:
Please give these indexers names so that when we convert to TypeScript it won't just be `a` and `b`. For example this one would be:

```suggestion
  attributeByClassIdAndNum: { [classId: string]: { [num: string]: PgAttribute } },
```

Reply:
cool, thanks